stf/model/prediction/lightgbm/quantile.py

```diff
@@ -0,0 +1,147 @@
+# SPDX-FileCopyrightText: 2017-2021 Alliander N.V. <korte.termijn.prognoses@alliander.com> # noqa E501>
```

Comment:
I think you accidentally also copied all the xgboost quantile models in the lightgbm folders and did not implement it. Could you remove these files?

Reply:
I'll change it, thanks!

Reply:
This file it still here I think.